server/dba/migrations/modules/transaction.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+let Promise = require('bluebird');
+
+let doTransaction = (tableName, queryInterface, operations) => {
+  return queryInterface.sequelize.transaction( (trx) => {
+
+    let moreOperations = addPreparations(tableName, operations);
+
+    return Promise.each(moreOperations, (operation) => {
+      operation.transaction = trx;
+      switch (operation.operation) {
+      case 'add': {
+        return queryInterface.addColumn(tableName, operation.field, {type: operation.type, allowNull: (operation.allowNull !== undefined ? operation.allowNull : true) }, {transaction: trx});
+      }
+      case 'remove': {
+        return queryInterface.removeColumn(tableName, operation.field, {transaction: trx});
+      }
+      case 'rename': {
+        return queryInterface.renameColumn(tableName, operation.field, operation.newField, {transaction: trx});
+      }
+      case 'change': {
+        return queryInterface.changeColumn(tableName, operation.field, operation.options, {transaction: trx});
+      }
+      case 'raw': {
```

Comment:
I will admit, I'm a little surprised that this is doesn't already exist within sequelize.